cms/templates/js/add-list-item.underscore

```diff
@@ -0,0 +1,3 @@
+<button class="action action-add">
```

Comment:
Arg. I did inline this in `list.js`, but the file must have been added back during a rebase. I'll remove it again...